### DISCUSSION TITLE
Creating Package Feed for An Application Using SDK build

### DISCUSSION THREAD

**Discussion by Hariom-avin** [OE4T/meta-tegra#922] [Q&A]
Hi,

I came to know about the package feed feature, through which we can update the application in the target image by using apt-get commands. I just read about it on following link.
 https://subscription.packtpub.com/book/virtualization-and-cloud/9781784395186/1/ch01lvl1sec21/setting-up-a-package-feed. 

This link at high level ask user to set up a server for build/tmp/deploy/deb repo and updating  /etc/apt/sources.list file at target with repo path.

I followed the steps and I am able to update the application on target using apt-get commands

For my work i use Yocto SDK for building the application software as a result i get application executable that i deploy in target using SCP command. 

So my question is, Can I also setup a package feed for my target if I use Yocto SDK ? If yes, Can you please guide me how can I set it up and update the application easily on the target using apt-get commands.

**Comment by madisongh** [comment]
I don't do package feeds in any of my projects, but my understanding is that you would need to have a recipe for building your application in your distro and publish the package in the feed as part of your distro builds.  The SDK is there to make the cross-compilation tools available for development and is not hooked into the build/deploy process.

Maybe the Extensible SDK that the YP tools support would help here?  I don't use it myself, so I don't really know. You might want to ask this on one of the Yocto Project mailing lists or in their IRC channel for ideas on simplifying the deployment workflow for applications under development.
